java_assignments/assignment8/Assignment8.java

```diff
@@ -0,0 +1,42 @@
+package assignments.assignment8;
```

Comment:
i think the assignment also asks us to check for NullPointer right. please check once.